assets/dev/js/admin/floating-elements/layout.js

```diff
@@ -0,0 +1,28 @@
+var NewFloatingElementsView = require( 'elementor-admin/floating-elements/view' );
+
+module.exports = elementorModules.common.views.modal.Layout.extend( {
```

Comment:
```suggestion
export default class extends elementorModules.common.views.modal.Layout {
```
Can this be an ES6 module?